### PR TITLE
Project cleanup

### PR DESCRIPTION
**Pull request by mevryn**

### CODE REVIEW

File: Assets/Scripts/appearance/PlayerAppearance/PlayerAppearance.cs [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+		}
+
+	}
+		
+	public void clothesChoose(int posision)
+	{
+		gameObject.transform.FindChild ("Boots").GetComponent<SpriteRenderer> ().sprite = chestChoosed[posision];
+		gameObject.transform.FindChild ("Pants").GetComponent<SpriteRenderer> ().sprite = pantsChoosed[posision];
+		gameObject.transform.FindChild ("Gloves").GetComponent<SpriteRenderer> ().sprite = glovesChoosed[posision];
+		gameObject.transform.FindChild ("Chest").GetComponent<SpriteRenderer> ().sprite = bootsChoosed[posision];
```
Comment (akomisarek) [inline Assets/Scripts/appearance/PlayerAppearance/PlayerAppearance.cs]: Bad design of dependant array. If 4 arrays are dependant on each other they should be extracted into separate class. What if the size is different? Think about it.

File: Assets/Scripts/appearance/PlayerAppearance/PlayerAppearance.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+
+	}
+
+	public int direction()
+	{
+		
+		if (Input.GetKeyDown (controls.instance.firstPlayerMoveRight))
+		{
+			return 2;
```
Comment (akomisarek) [inline Assets/Scripts/appearance/PlayerAppearance/PlayerAppearance.cs]: What do those numbers mean? Create some enumaration types or static. Never should have 0 / 1 / 2 / 3 returned from the code it is meaningles. Use something like Direction.UP Direction.DOWN etc. A lot better design.

File: Assets/Scripts/appearance/PlayerAppearance/PlayerAppearance.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+			return directionOfLastMovement;
+		}
+		
+	}
+
+	
+	public void movementSpriteChanger()
+	{
+		
+			movementAppearanceScripts.movementSpriteChanger(maleSprites,direction(),gameObject.transform.FindChild ("Body").GetComponent<SpriteRenderer> ());
```
Comment (akomisarek) [inline Assets/Scripts/appearance/PlayerAppearance/PlayerAppearance.cs]: The formatting is not consistent. Those tabs looks ugly.

File: Assets/Scripts/appearance/PlayerAppearance/PlayerAppearance.cs [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+			movementAppearanceScripts.movementSpriteChanger (pantsChoosed, direction(), gameObject.transform.FindChild ("Pants").GetComponent<SpriteRenderer> ());
+			movementAppearanceScripts.movementSpriteChanger (bootsChoosed, direction(), gameObject.transform.FindChild ("Boots").GetComponent<SpriteRenderer> ());
+			movementAppearanceScripts.movementSpriteChanger (chestChoosed, direction(), gameObject.transform.FindChild ("Chest").GetComponent<SpriteRenderer> ());
+			directionOfLastMovement = direction ();
+
+	}
+
+	void Start ()
+	{
+		sex = 0;
```
Comment (akomisarek) [inline Assets/Scripts/appearance/PlayerAppearance/PlayerAppearance.cs]: Again learn about enums type. It is really hard to follow 0 / 1 around the code.

File: Assets/Scripts/movement/Player/Player.cs [hunk 5]
```diff
@@ -0,0 +1,36 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class Player : MonoBehaviour {
+
+	public float moveSpeed;
+	// Use this for initialization
+	[HideInInspector]
+	public Movement movement;
```
Comment (akomisarek) [inline Assets/Scripts/movement/Player/Player.cs]: Why this is not private then? If not needed by inspector?

File: Assets/Scripts/movement/Player/Player.cs [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+	
+	// Update is called once per frame
+	void FixedUpdate () {
+		playerMovement ();
+	
+	}
+
+	void Update()
+	{
```
Comment (akomisarek) [inline Assets/Scripts/movement/Player/Player.cs]: Decide upon curly brackets. They should be consistent
either {
}
or 
{
}.

The code is clumsy and doesn't look nice.

File: obj/Debug/Assembly-CSharp.csproj.FileListAbsolute.txt [hunk 7]
```diff
@@ -21,7 +21,6 @@ I:\RogueLike\RogueLike\obj\Debug\Assembly-CSharp.dll
 I:\RogueLike\RogueLike\obj\Debug\Assembly-CSharp.pdb
 <<<<<<< HEAD
 =======
-<<<<<<< HEAD
 >>>>>>> e91d37ba24fbd2cabe2204d6d84e4668b03cc45f
 F:\programowanie\RougeLike\Temp\bin\Debug\Assembly-CSharp.dll
```
Comment (akomisarek) [inline obj/Debug/Assembly-CSharp.csproj.FileListAbsolute.txt]: Conflict markers. Is this file even required? If not it should be in .gitignore.

Comment (mevryn) [inline obj/Debug/Assembly-CSharp.csproj.FileListAbsolute.txt]: obj folder is in the .gitignore file

File: Assets/Scripts/movement/Player/Player.cs (hunk 5 above)
Comment (mevryn) [inline Assets/Scripts/movement/Player/Player.cs]: Becouse i added it in the inspector as and specified Movement Object Script, right ? And later I hid it in Inspector.
